scripts/sct_run_batch.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# Wrapper to processing scripts, which loops across subjects. Data should be
+# organized according to the BIDS structure:
+# https://github.com/sct-pipeline/spine_generic#file-structure
+#
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Chris Hammill
+#
+# About the license: see the file LICENSE.TXT
+##############################################################################
+
+from __future__ import absolute_import
+
+import os
+import sys
+import argparse
+import numpy as np
+import multiprocessing
+import subprocess
+import re
+import time
+
+import spinalcordtoolbox.image as msct_image
+from spinalcordtoolbox.image import Image, concat_data
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+
+parser = argparse.ArgumentParser(
+  description = 'Wrapper to processing scripts, which loops across subjects. Data should be '
```

Comment:
BIDS structure is not mandatory for the script to run. In fact, the processing does not follow BIDS convention (where outputs should be placed under a separate derivatives/ folder). So I would relax this statement by just saying something like: "All subjects' folder should be placed under PATH_DATA and the file naming should be consistent across subjects. We recommend to use the BIDS convention (https://bids.neuroimaging.io/)"

Reply:
I chose a wording similar to your suggestion but with no explicit reference to `PATH_DATA`, from the perspective of the python program it's `--path-data` but I think the key is that it's all in one directory. We can obviously change this to be closer to your original wording if you want.

Reply:
@jcohenadad Can you confirm if the new wording is ok?